### ISSUE TITLE
Behavior of --with-attribute has changed?

### ISSUE THREAD

**Issue by myd7349** [dotnet/ClangSharp#565]
When I was using ClangSharpPInvokeGenerator to process [edflib.h](https://gitlab.com/Teuniz/EDFlib/-/blob/master/edflib.h?ref_type=heads), I noticed that different versions of ClangSharpPInvokeGenerator handle the `--with-attribute` option differently.

My response file:

```
--config
compatible-codegen
generate-macro-bindings
multi-file
windows-types
--file
./edflib.h
--libraryPath
EDFlib
--language
c++
--methodClassName
EDF
--namespace
SharpEDFlib.Interop
--output
./SharpEDFlib/Interop
--with-attribute
edf_set_gender=Obsolete("Use edf_set_sex instead.")
edfwrite_annotation_utf8=Obsolete("Use edfwrite_annotation_utf8_hr instead.")
edfwrite_annotation_latin1=Obsolete("Use edfwrite_annotation_latin1_hr instead.")
--with-namespace
edf_set_gender=System
edfwrite_annotation_utf8=System
edfwrite_annotation_latin1=System
```

Here, I plan to use the `--with-attribute` option to add the `Obsolete` attribute to three deprecated interfaces.

With ClangSharpPInvokeGenerator 14.0.0:

```
        [DllImport("EDFlib", CallingConvention = CallingConvention.Cdecl, ExactSpelling = true)]
        [Obsolete("Use edf_set_sex instead.")]
        public static extern int edf_set_gender(int handle, int gender);
```

With latest version of ClangSharpPInvokeGenerator:

```
        [DllImport("EDFlib", CallingConvention = CallingConvention.Cdecl, ExactSpelling = true)]
        [Obsolete(Use edf_set_sex instead.)]
        public static extern int edf_set_gender(int handle, int gender);
```

Is this change as expected or do I miss something?

Thank you!

**Comment by tannergooding** [comment]
This is a bug in the latest version of System.CommandLine, it also no longer correctly handles lines with spaces in them

I need to revert back to the previous version

**Comment by myd7349** [comment]
Confirm fixed in 18.1.0.1.
